### DISCUSSION TITLE
Vibration not working over bluetooth, but working through usb

### DISCUSSION THREAD

**Discussion by ghost** [nefarius/DsHidMini#97] [Q&A]
BthPS3 1.3.127.0
DS4windows 3.0.5
Dshidmini v1.4.222.0
Intel wireless bluetooth USB\VID_8087&PID_0A2B&REV_0001

After following the main guide and setting everything up, I noticed, using MightyHIDTest, that the rumble funtion does not work over bluetoth for my ps3 controller, is this a resurgence of #4? also apparently according to Dshidmini my controller is not genuine.
![IMG_20210518_225723_1_2](https://user-images.githubusercontent.com/84419163/118728726-90106f00-b82c-11eb-94a5-7356182ea714.jpg)

**Comment by ghost** [comment]
Umm, hey?, I still have this problem, and need assistance

**Comment by Kanuan** [comment]
Oh, forgot about this. 
It's not related to the issue you linked, that was (supposed to be) fixed for good.
Can you send a screenshot of your DSHMC when connected over BT?
Besides that, join the server's discord so it's easier to test on this

**Comment by ghost** [comment]
Alright, will upload once I get to my laptop, do I send it here or to the discord server?

**Comment by Kanuan** [comment]
send it here to keep it registered, then we continue on the server

**Comment by ghost** [comment]
![Screenshot (23)](https://user-images.githubusercontent.com/84419163/119259062-5771ec00-bbc4-11eb-8cac-4f8a43ff21c1.png)
![Screenshot (24)](https://user-images.githubusercontent.com/84419163/119259065-593baf80-bbc4-11eb-96db-72adae462010.png)

Done!

**Comment by Kanuan** [comment]
Just an update on what followed on the server:
- The controller correctly updates LED states over bluetooth, so it is properly receiving output reports, but rumble commands are ignored nonetheless
- The OP can't test if the controller actually rumbles over bluetooth when connected to a real PS3
- Though it is weird for a controller to rumble over USB but not via BT, since it can't be confirmed if it's an issue with the controller or with DsHidMini the whole troubleshooting has been put on hold until the user has managed to properly confirm the issue doesn't exist with a real PS3

**Comment by AhmadMoemen** [comment]
Any updates on this issue? Because I have the same controller as OP and still have the same issue on the latest DsHidMini and BthPS3 versions...
Also LED doesn't get updated, and rumble/vibration doesn't work as well (both through Bluetooth)
On USB it works fine, and btw motion doesn't work in all cases so I don't mind it anyways.

**Reply by Kanuan** [reply]
Basically, we couldn't confirm the the other user's controller had working rumble AT ALL with his controller. They didn't have a real PS3 to test so it was possible it just didn't work via Bluetooth in any situation.

You can try testing with ScpToolKit v1.17. If it works wirelessly with it we would at least have some a lead on this

**Reply by AhmadMoemen** [reply]
It just disconnects right away?

**Reply by AhmadMoemen** [reply]
Runs at a very short duration with bluetooth then disconnects in all sudden, also LED commands are not received either...
Not sure how I can log this sadly....

**Reply by AhmadMoemen** [reply]
[ScpDsxService_DESKTOP-V3752PR$.log.xml.txt](https://github.com/nefarius/DsHidMini/files/12666624/ScpDsxService_DESKTOP-V3752PR.log.xml.txt)
Here's a log :D

**Reply by Kanuan** [reply]
Is the dongle using ScpToolKit's drivers for BT adapter? Does a ScpToolKit notification appear when the controller is connected wirelessly?

If the notification appears then the controller is connecting through ScpToolKit's BT adapter and failing to maintain the connection anyway, so it doesn't work and we have no lead on investigating this

**Reply by AhmadMoemen** [reply]
yes, and also I checked the Scp Monitor, it shows the BT host address and everything, I'll try a restart I guess?
I didn't properly read the log yet so lemme check if there's something suspicious...
I tried disabling rumble/disabling LED status but those didn't maintain the connection either.

**Reply by AhmadMoemen** [reply]
I'll try this version... https://github.com/nefarius/ScpToolkit/releases/tag/v1.6.133.15324

**Reply by AhmadMoemen** [reply]
It's maintaining connection this time! But vibration isn't working over Bluetooth, nor the LED status...

**Reply by Kanuan** [reply]
Yeah. No idea on what to do then. It's basically impossible to try figuring out why it's not working.

**Reply by AhmadMoemen** [reply]
Indeed, I wish I could figure this out myself XD
Sadly my real PS3 is glod due to RSX :(
If I managed to test the controller on a real PS3 I'll keep you updated.
And if there's anything you can suggest I'm all in for testing.
